packages/plugins/rubygem-git/rubygem-git.spec

```diff
@@ -5,8 +5,8 @@
 
 Summary:        A package for using Git in Ruby code
 Name:           %{?scl_prefix}rubygem-%{gem_name}
-Version:        1.2.5
-Release:        9%{?dist}
+Version:        1.5.0
+Release:        1%{?dist}
 Group:          Development/Languages
 License:        MIT
 URL:            http://rubyforge.org/projects/git/
```

Comment:
Looks like this URL is dead. Could you update it as well? And the Source0 link also doesn't look up to date anymore.